packages/ui/src/molecules/Dropdown/DropdownMenu.tsx

```diff
@@ -105,8 +107,6 @@ const DropdownMenu = ({
 
     event.key === 'End' && handleEndPress()
 
-    event.key === 'Tab' && event.preventDefault()
```

Comment:
IS because "Tab" just called the preventDefault, and now the preventDefault is called with any key, except the "Enter", because is needed that the "Enter" execute his default event